server/src/test/java/org/elasticsearch/index/mapper/TextFieldTypeTests.java

```diff
@@ -38,47 +38,16 @@
 import org.elasticsearch.common.lucene.BytesRefs;
 import org.elasticsearch.common.unit.Fuzziness;
 import org.elasticsearch.index.mapper.TextFieldMapper.TextFieldType;
-import org.junit.Before;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import java.util.Map;
 
 import static org.apache.lucene.search.MultiTermQuery.CONSTANT_SCORE_REWRITE;
 import static org.hamcrest.Matchers.equalTo;
 
-public class TextFieldTypeTests extends FieldTypeTestCase<TextFieldType> {
-
-    @Before
```

Comment:
We check that these modifications block merging in TextFieldMapperTests, so we can safely remove them here - this is just checking the `equals` implementation, which is now gone.